__init__.py

```diff
@@ -0,0 +1,140 @@
+from neon_utils.skills.neon_skill import NeonSkill, LOG
+import os
+import json
+from adapt.intent import IntentBuilder
+# from neon_utils.instruction_check import Check
+
+class InstructionsSkill(NeonSkill):
+
+    def __init__(self):
+
+        super(InstructionsSkill, self).__init__(name="InstructionsSkill")
+        basepath = os.path.dirname(os.path.realpath(__file__))
+        json_path = basepath+'/scripts/en/demo2_en.jsonl'
+        self.json_path = json_path
+        self.Check = None
+        self.json_path = json_path
+        self.answer_list = []
+        self.question_id = '1'
+        self.words_from_prev_answer = ''
+
+    #alternative intent parsing 
+    
+    # def initialize(self):
+    #     start_instructions_intent = IntentBuilder('run_instructions.intent').require("instructions_keyword").optionally('neon').build()
+    #     # self.register_intent_file(start_instructions_intent, self.start_instructions_intent)
+
+    def initialize(self):
+        self.register_intent_file("run_instructions.intent", self.handle_instructions)
+
+        # When first run or demo prompt not dismissed, wait for load and prompt user
+        if self.settings['prompt_on_start'] and not self.server:
+            self.bus.once('mycroft.ready', self._start_instructions_prompt)
+
+    def start_instructions_intent(self, message):
```

Comment:
Preferred intent syntax is to use decorators instead of registering in initialize. Makes no difference in execution, decorators are just more consistent with other skills

```python
from mycroft.skills.core import intent_file_handler

@intent_file_handler("run_instructions.intent")
def start_instructions_intent(self, message):
```

Reply:
In this implementation initialize() isn't needed?

Reply:
I have added that but now tests don't pass, I don't understand the reason

Reply:
Sorry, I broke unit tests with this change: https://github.com/NeonGeckoCom/neon-skill-utils/commit/b020c7a432e5f3d3bda6b5c0376a649ff9807758#diff-571deb65366076f475dd1ba81c4974ef1cb54d582d5d82c9f0eb1767069d5898L73-R75

In the `setUp` method, `self.local_config` needs to change to `self._local_config`